docker-compose.yml

```diff
@@ -17,12 +18,13 @@ services:
       - .:/usr/src/app
     ports:
       - "3000:3000"
-    links:
```

Comment:
`links` and `depends_on` are not 1:1. Links was a shortcut for `network` more or less, and KINDA did `depends_on` to some degree (which doesn't 100% work either, but that's a whole other issue). Generally when you remove `links`, you add `network`, because `network` does what `links` did but it also sets up other good stuff.

Pretty sure things will work as is, but you might take a look at the `network` command, and see if there is any benefit to using it. I've been teaching some friends golang and you can see a very basic use of it here: https://github.com/DarthHater/hello-world/blob/TestBranch/docker-compose.yml#L27

`depends_on` in short will only wait for the container to start, for the record. If you truly want to wait till the DB is up you gotta kinda hand roll a `bash` script or something akin that can ping something, and then return true once it's ready to go.

Reply:
I shared this mostly because I can't see the `network` stuff, for the record, and your branch has it in the name :)

Reply:
the nice thing about docker-compose here is that it will automatically do the network stuff if you don't declare one! So it infers that everything is in a shared network and automagically sets it up as part of `docker-compose (run|up)` which is pretty handy.